### PR TITLE
Moove/Butler operator integration

### PR DESCRIPTION
**Pull request by lucasbfernandes**

### CODE REVIEW

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt [hunk 1]
```diff
@@ -42,6 +42,7 @@ enum class MooveErrorCode(val key: String) {
     WORKSPACE_GIT_CONFIGURATION_IS_MISSING("workspace.git.configuration.is.missing"),
     WORKSPACE_DOCKER_REGISTRY_CONFIGURATION_IS_MISSING("workspace.docker.registry.configuration.is.missing"),
     WORKSPACE_CD_CONFIGURATION_IS_MISSING("workspace.cd.configuration.is.missing"),
```
Comment (thallesfreitaszup) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt:44]: this error code is not necessary anymore

**Review by thallesfreitaszup — CHANGES_REQUESTED** [review]
i think that the cd configuration should be removed from database as well

**Review by leandrorodrigueszup — APPROVED** [review]

File: moove/application/src/main/kotlin/io/charlescd/moove/application/workspace/request/PatchWorkspaceRequest.kt [hunk 2]
```diff
@@ -31,7 +31,8 @@ data class PatchWorkspaceRequest(override val patches: List<PatchOperation>) : B
             "/registryConfigurationId",
             "/cdConfigurationId",
```
[the remark this answers is not in the document]
Comment (lucasbfernandes) [reply moove/application/src/main/kotlin/io/charlescd/moove/application/workspace/request/PatchWorkspaceRequest.kt]: done 👍

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt (hunk 1 above)
Comment (lucasbfernandes) [reply moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt]: done 👍

**Review by thallesfreitaszup — APPROVED** [review]

**Pull request merged**